apps.awesim.org/apps/bc_desktop/vdi-owens.yml

```diff
@@ -1,17 +1,13 @@
 ---
 title: "Owens VDI"
 description: |
-  This app will launch an interactive desktop on a **shared node** that we have
-  termed a Virtual Desktop Interface (VDI). This is analogous to an interactive
-  login node as you will be sharing it with multiple users and you *should* be
-  provisioned a desktop nearly immediately.
+  This app will launch an interactive desktop with one core which could be shared. It
+  is small environment for general-purpose tasks like sandboxing, prototyping,
+  viewing results and editing files.
```

Comment:
After seeing side by side with what was originally there I'd suggest the second sentence be modified like this:

It is a small environment for lightweight tasks (similar to a login node) such as accessing & viewing files, submitting jobs, compiling code, and running visualization software. You should be provisioned a desktop nearly immediately.

That's from the bullet points of the original list, which I think more specifically addresses the types of things our audience does on login nodes.

Reply:
OK